FirebaseRemoteConfig/Sources/RCNConfigConstants.h

```diff
@@ -19,13 +19,6 @@
 #define RCN_SEC_PER_MIN 60
 #define RCN_MSEC_PER_SEC 1000
 
-/// Remote Config SDK internal version that is different than
-/// FIRRemoteConfigPodVersion. This is for config server to track down iOS
```

Comment:
Based on the description, it seems RC backend does track a separate version that is different than the pod version. I would double check with RC backend team to make sure this is no longer needed. 
Also considering this field seems expect a number, e.g. for v1.2 it's 10200. I wonder if this field should be passed in at all now that we are passing in a string "1.2.0"

Reply:
Checked in the backend and we don't use `sdk_version` anywhere except for logging/metrics, and we recently fixed our logging to handle this field as a string (updated bug with details)

Reply:
lol looks like this should be updated for a long while. Thanks for doing it!